index.md

```diff
@@ -171,12 +171,16 @@ This power is only available in 5/6 player games, but the version which is prese
 ### The Flip-Flop
 Early voting can be unproportionally counted on compared to other votes. In the first round of voting, Fascists have the advantage and it is likely that liberals will vote against, wasting a potential government. Flip-flopping between games can help mitigate this due to balancing skeptical Liberals to optimistic ones. WARNING: This has a side effect of unpredictable player meta; players may see you as untrustworthy because they cannot reason about your initial vote - especially if you were next in line to be President.
 
+### The Slow Burn
 Liberal investigating a fascist, claims they are a liberal, sits on that information for a while and then revealing that person is a fascist later.
 
+### The Maverick Investigator
 Liberal President force their Chancellor to play a fascist to receive the investigation power (only available in 6+ player games) to investigate their Chancellor. An extension to this play happens when the Liberal President find their Chancellor is a Fascist but informs the other players that they are Liberal. The reaction the Fascist Chancellor may reveal if the Chancellor is a Fascist or Hitler. Hitler would may react to the move, whereas a Fascist may be startled by answer.
 
+### The Boy that Cried Wolf
 Claiming to have accidentally played the wrong (i.e., fascist) policy.
```

Comment:
I think it's best to not mention our names outside of the contributors list so that it's more consumable by those outside of our group.